op-e2e/actions/l2_engine_api.go

```diff
@@ -42,13 +42,13 @@ func computePayloadId(headBlockHash common.Hash, params *eth.PayloadAttributes)
 	_ = binary.Write(hasher, binary.BigEndian, params.Timestamp)
```

Comment:
Extending the compute payload ID method is kind of weird, since the engine API spec defines it to be a random ID, but geth still uses a legacy method of hashing the payload attributes. I opened ENG-2976 to fix this upstream. Then we won't need this diff, nor would we have to have all this complexity in the action testing.

Reply:
I didn't realize that hashing was a legacy way of computing the payload id, is the preferred way just getting random bytes?

Reply:
yes